### ISSUE TITLE
List of notifications height

### ISSUE THREAD

**Issue by mindrunner** [ErikReider/SwayNotificationCenter#177]
My notification stack seems to have a height limit which does not really look nice:

![image](https://user-images.githubusercontent.com/1413542/200288240-29ac49be-4475-4257-bd84-b87d6ac85c5f.png)

After about 6 notifications, it starts having a scrollbar instead of using the whole screen height.
Opening the notification-center works as expected (full screen height).
Am I missing some configuration here?

```
{
  "$schema": "/etc/xdg/swaync/configSchema.json",
  "positionX": "center",
  "positionY": "top",
  "control-center-margin-top": 15,
  "control-center-margin-bottom": 15,
  "control-center-margin-right": 0,
  "control-center-margin-left": 0,
  "notification-icon-size": 64,
  "notification-body-image-height": 100,
  "notification-body-image-width": 200,
  "timeout": 10,
  "timeout-low": 5,
  "timeout-critical": 0,
  "fit-to-screen": true,
  "control-center-width": 500,
  "notification-window-width": 500,
  "keyboard-shortcuts": true,
  "image-visibility": "when-available",
  "transition-time": 200,
  "hide-on-clear": true,
  "hide-on-action": true,
  "script-fail-notify": true,
  "notification-visibility": {
    "ignore-spotify": {
      "state": "ignored",
      "urgency": "Low",
      "app-name": "Spotify"
    }
  },
  "widgets": [
    "notifications",
    "mpris",
    "title",
    "dnd"
  ],
  "widget-config": {
    "title": {
      "text": "",
      "clear-all-button": true,
      "button-text": "Clear All"
    },
    "dnd": {
      "text": "Do Not Disturb"
    },
    "label": {
      "max-lines": 5,
      "text": "Label Text"
    },
    "mpris": {
      "image-size": 96,
      "image-radius": 12
    }
  }
}
```

**Comment by ErikReider** [comment]
Duplicate of #174

**Comment by ErikReider** [comment]
Fixed by #561
